docs/reference/text-structure/apis/find-structure.asciidoc

```diff
@@ -60,67 +60,67 @@ chosen.
 == {api-query-parms-title}
 
 `charset`::
-(Optional, string) The file's character set. It must be a character set that is 
+(Optional, string) The file's character set. It must be a character set that is
```

Comment:
This line highlights that there are loads of places in these docs that refer to "the file".  Now the endpoint is _text_ structure, they should probably be reworded to refer to "the text" (which may require more than just find and replace to sound good).

But this can be a followup item for a different PR.  Best not to mix with this one or it will obscure the main point of this one.

Reply:
yeah, I messaged @lcawl about something similar on the docs work. I do think changing to `text` where appropriate should be done.